apps/google-analytics-4/frontend/src/components/sidebar/SidebarAnalytics.tsx

```diff
@@ -0,0 +1,46 @@
+import { useAutoResizer, useFieldValue } from '@contentful/react-apps-toolkit';
+import ChartFooter from 'components/sidebar/ChartFooter';
+import ChartHeader from 'components/sidebar/ChartHeader';
+import { useEffect, useState } from 'react';
+import { RunReportResponse } from 'types';
+import { config } from '../../config';
+
+export default function SidebarAnalytics() {
+  const [runReportResponse, setRunReportResponse] = useState<RunReportResponse | undefined>();
```

Comment:
I have an idea here: so if you set this value like
`useState<RunReportResponse>({} as RunReportResponse);`
then you can do the following on lines 37 and 38.